README.md

```diff
@@ -1,18 +1,50 @@
-# Confidential Container Tools and Components for the tenant side
+# Trusted Components for Attestation and Secret Management
 
 [![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fconfidential-containers%2Fkbs.svg?type=shield)](https://app.fossa.com/projects/git%2Bgithub.com%2Fconfidential-containers%2Fkbs?ref=badge_shield)
 
-This repository includes tools and components for the tenant side of confidential containers.
+This repository contains tools and components for attesting confidential guests and providing secrets to them.
+These components typically operate on behalf of the guest owner and interact remotely with 
+[guest components](https://github.com/confidential-containers/guest-components).
+
+These components were developed for the Confidential Containers project, but can be used with a wide variety
+of applications and hardware platforms.
 
 ## 
 
 [Key Broker Service](kbs)
-The Confidential Containers Key Broker Service (KBS) is a remote server which facilitates remote attestation, which is the reference implementation of [Relying Party](https://www.ietf.org/archive/id/draft-ietf-rats-architecture-22.html)
-and [Verifier](https://www.ietf.org/archive/id/draft-ietf-rats-architecture-22.html) in [RATS](https://datatracker.ietf.org/doc/draft-ietf-rats-architecture/)
-role terminology.
+The KBS is a server that facilitates remote attestation and secret delivery.
+It's role is similar to that of the [Relying Party](https://www.ietf.org/archive/id/draft-ietf-rats-architecture-22.html#name-relying-party)
+in the RATS model.
 
 [Attestation Service](attestation-service)
-Attestation Service (AS for short) is a general function set that can verify TEE evidence.
+The AS verifies TEE evidence.
+In the RATS model this is a [Verifier](https://www.ietf.org/archive/id/draft-ietf-rats-architecture-22.html#name-verifier)
+
+[Reference Value Provider Service](attestation-service/rvps)
+The RVPS manages reference values used to verify TEE evidence. 
+This is [mentioned](https://www.ietf.org/archive/id/draft-ietf-rats-architecture-22.html#name-endorser-reference-value-pr)
+in the RATS model as well.
+
+[Client Tool](kbs/tools/client)
+This is a simple tool which can be used to test or configure the KBS and AS.
+
+##
+
+These components are flexible and can be deployed in several different configurations.
+This figure shows one common way to deploy these components in conjunction with certain guest components.
+
+<div align=center>
+
+![](kbs/docs/pictures/cluster.svg)
+
+</div>
```

Comment:
I'm thinking if skopeo and encrypted container image is really needed. We can just mentioned loading of keys in the right path to keep it simple

Reply:
Ok, I replaced the diagram with a fancy mermaid one. I am using a flowchart, which has some limitations in terms of layout. Mermaid also supports more complex C4 diagrams, but only as an experimental feature. What do you think about the flow chart? cc @Xynnn007

Reply:
Looks good to me.